common/src/main/java/com/discordsrv/common/AbstractDiscordSRV.java

```diff
@@ -769,7 +765,27 @@ protected List<ReloadResult> reload(Set<ReloadFlag> flags, boolean initial) thro
                 messagesConfigManager().reload(configUpgrade, anyMissingOptions, backupPath);
 
                 if (anyMissingOptions.get()) {
-                    logger().info("Use \"/discordsrv reload config_upgrade\" to write the latest configuration");
+                    if (config().automaticConfigurationUpgrade) {
+                        logger().info("Some configuration options are missing, attempting to upgrade configuration...");
+
+                        if(backupPath == null) backupPath = generateBackupPath();
+                        AtomicBoolean stillMissingOptions = new AtomicBoolean(false);
+
+                        connectionConfigManager().reload(true, stillMissingOptions, backupPath);
+                        configManager().reload(true, stillMissingOptions, backupPath);
+                        messagesConfigManager().reload(true, stillMissingOptions, backupPath);
+
+                        if (stillMissingOptions.get()) {
+                            logger().warning("Attempted to upgrade configuration automatically, but some options are still missing.");
+                        } else {
+                            logger().info("Configuration successfully upgraded");
+                        }
+                    } else if (configUpgrade) {
+                        logger().warning("Attempted to upgrade configuration by reload command, but some options are still missing.");
+                    } else {
+                        logger().info("Use \"/discordsrv reload config_upgrade\" to write the latest configuration");
+                        logger().info("Or Set \"automatic-configuration-upgrade\" to true in the config to automatically upgrade the configuration on startup");
```

Comment:
```suggestion
                        logger().info("Or set \"automatic-configuration-upgrade\" to true in the config to automatically upgrade the configuration on startup");
```
As is this runs on every reload not just on startup, I'm fine with either but this log line should reflect the actual behaviour